### PR TITLE
[tests-only] Adjust userTest interactions for oCIS

### PR DESCRIPTION
**Pull request by saw-jan**
part of https://github.com/owncloud/owncloud-sdk/issues/644 and https://github.com/owncloud/owncloud-sdk/issues/747

adjust interactions for oCIS:
- userTest.js

### CODE REVIEW

File: tests/userTest.js [hunk 1]
```diff
@@ -338,6 +359,8 @@ describe('Main: Currently testing user management,', function () {
     })
   })
 
+  // [OCIS] subadmin endpoint not implemented
+  // https://github.com/owncloud/product/issues/289
```
[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/userTest.js:369]: IMO, that comment will eliminate the need to go through the links inorder to know what is the issue

[the remark this answers is not in the document]
Comment (saw-jan) [reply tests/userTest.js:369]: IMO, that comment will eliminate the need to go through the links inorder to know what is the issue

**Review by swoichha — COMMENTED** [review]
lgtm

**Review by jasson99 — COMMENTED** [review]
LGTM. Please rebase

**Review by kiranparajuli589 — COMMENTED** [review]
lgtm :+1:

**Review by Talank — COMMENTED** [review]
LGTM

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by individual-it — APPROVED** [review]

**Pull request merged**